moped-editor/src/views/projects/signalProjectTable/SignalProjectTable.js

```diff
@@ -0,0 +1,383 @@
+import React from "react";
+import { useQuery, useMutation } from "@apollo/client";
+import {
+  CardContent,
+  CircularProgress,
+  Grid,
+  TextField,
+  Typography,
+  makeStyles,
+} from "@material-ui/core";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import MaterialTable, { MTableEditRow, MTableEditField } from "material-table";
+
+import Page from "src/components/Page";
+import typography from "../../../theme/typography";
+import {
+  SIGNAL_PROJECTS_QUERY,
+  UPDATE_SIGNAL_PROJECT,
+} from "../../../queries/signals";
+import { PROJECT_UPDATE_SPONSOR } from "../../../queries/project";
+import { PAGING_DEFAULT_COUNT } from "../../../constants/tables";
+import RenderFieldLink from "./RenderFieldLink";
+
+const useStyles = makeStyles({
+  signalsTable: {
+    "& .MuiTableCell-root": {
+      // override the default padding of 16px
+      padding: "14px",
+    },
+  },
+});
+
+const SignalProjectTable = () => {
+  const classes = useStyles();
+  const { loading, error, data, refetch } = useQuery(SIGNAL_PROJECTS_QUERY, {
+    fetchPolicy: "no-cache",
+  });
+  const typographyStyle = {
+    fontFamily: typography.fontFamily,
+    fontSize: "14px",
+  };
+
+  const [updateSignalProject] = useMutation(UPDATE_SIGNAL_PROJECT);
+  const [updateProjectSponsor] = useMutation(PROJECT_UPDATE_SPONSOR);
+
+  if (error) {
+    console.log(error);
+  }
+  if (loading || !data) {
+    return <CircularProgress />;
+  }
+
+  const entityList = data?.moped_entity ?? [];
+
+  /**
+   * Build data needed in Signals Material Table
+   */
+  data.moped_project.forEach(project => {
+    // project status update equivalent to most recent project note
+    project["status_update"] = "";
+    if (project?.moped_proj_notes?.length) {
+      const note = project.moped_proj_notes[0]["project_note"];
+      // Remove any HTML tags
+      project["status_update"] = note
+        ? String(note).replace(/(<([^>]+)>)/gi, "")
```

Comment:
I think the idea of stripping out HTML tags is very important, and the code here should get anything that is bounded by `<>`s, including nested closing tags. I'm not sure what would happen here with a double `<<` though. 

This is just me throwing out an idea -- HTML detection and cleanup is one of those things like timezones that is a rabbit hole that can be run down without ever finding the bottom. Would we may want to consider out-sourcing that task to a library whose only purpose is to do that and has a test suite? I found [this library, sanitize-html](https://github.com/apostrophecms/sanitize-html), which may fit the bill if we wanted to go that route at some point.

The thought reminds me of [Little Bobby Drop Tables](https://xkcd.com/327/). :)

Reply:
+1 and if sanitizing is the goal y'all have [already implemented it](https://github.com/cityofaustin/atd-moped/blob/235604e270c7423c359d7b20ef7f1ec12fefcd72/moped-editor/src/views/projects/projectView/ProjectComments.js) in project comments :)

Reply:
I'll use the project comments sanitizer, I copied the regex from the code Sergio wrote here (https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js#L49) to ensure the status would render the same in both places, but the library sanitizer will handle it better.